### ISSUE TITLE
Transformers 4.0.0 availability

### ISSUE THREAD

**Issue by xirect** [amaiya/ktrain#297]
Hi!

What an amazing package. I was trying to use some Longformer models with the huggingface API but I came to the problem that many configs aren't supported in Transformers < 4.0.0 as required by KTrain. Any idea when it will be supported?

Also another question, will there be support for PyTorch models too?

**Comment by amaiya** [comment]
The current version of **ktrain** should support `transformers>=4.0.0`.  All tests passed the last time I checked.  However, we are temporarily pinning to `transformers>3.1,<4.0`.  The reason is that 4.0.x completely shuffled around the code base, and this seems to break the loading of `Preprocessor` instances created with older versions of **transformers** which may cause problems for some folks.  We will upgrade to 4.0.x eventually but it may have to be a breaking change.   If you want to try `transformers>=4.0` now, you can do `pip install -U transformers` **after** installing **ktrain** and ignore the ERROR.

**ktrain** can already load PyTorch models and uses `from_pt` argument to `from_pretrained` when loading normally fails.

Finally, the Longformer model (using TensorFlow at least) does not seem to currently work in 4.0.x.   The error returned in **ktrain** is the same error returned when using **transformers** directly (without using **ktrain**).  So, it seems like either 1) a bug in **transforrmers** or 2) the Longformer model needs to be processed differently somehow and the location of the documentation for this has not yet been found by me or doesn't exist  yet.

**Comment by xirect** [comment]
Thank you for the clear explanation!
